package.json

```diff
@@ -2,7 +2,9 @@
   "name": "react-onclickoutside",
   "version": "5.11.1",
   "description": "An onClickOutside wrapper for React components",
-  "main": "index.js",
+  "main": "./lib/index.js",
```

Comment:
rather than this, let's have a `src` dir that contains the code, and have it rollup to `./index.js`, then keep everything the same: as a umd module everything should just load that index.js